.goreleaser.yml

```diff
@@ -25,6 +26,19 @@ builds:
       - amd64
       - arm64
 
+  - id: temporal-doc-gen
```

Comment:
Can we make sure this isn't added to the release archive that users consume please?

Reply:
done